CMakeLists.txt

```diff
@@ -4,17 +4,6 @@ project(rqt_service_caller)
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_python REQUIRED)
 find_package(rqt_py_common REQUIRED)
```

Comment:
This package is not a build dependency and therefore not available at build time. See #8.